test/spec/spec.form.js

```diff
@@ -515,14 +514,162 @@ describe('Form Controller', function () {
         });
 
         it('redirects to `next` page', function () {
-            form.successHandler(req, res);
-            res.redirect.should.have.been.calledWith('/success');
+            form.getNextStep(req, res).should.be.equal('/next-page');
         });
 
         it('prefixes redirect url with req.baseUrl', function () {
             req.baseUrl = '/base';
-            form.successHandler(req, res);
-            res.redirect.should.have.been.calledWith('/base/success');
+            form.getNextStep(req, res).should.be.equal('/base/next-page');
+        });
+
+        describe('with a fork', function () {
+
+            describe('when the condition config is met', function () {
+                it('the next step is the fork target', function () {
```

Comment:
Do we need a `describe` around one `it`? Reading 'it the next step is the fork target' doesn't make sense.